pyproject.toml

```diff
@@ -60,14 +60,14 @@ extra-dependencies = [
 ]
 
 [tool.hatch.envs.test.scripts]
-unit = 'pytest --cov-report xml:coverage.xml --cov="haystack_experimental" -m "not integration" {args:test}'
-integration = 'pytest --reruns 3 --reruns-delay 60 -x --maxfail=5 -m "integration" {args:test}'
+unit = 'pytest --cov-report xml:coverage.xml --cov="haystack_experimental" -m "not integration and not unstable" {args:test}'
+integration = 'pytest --reruns 3 --reruns-delay 60 -x --maxfail=5 -m "integration and not unstable" {args:test}'
```

Comment:
Let's not modify the hatch command to directly exclude these tests - this excludes those tests on non-CI hosts as well. Instead, let's pass an argument that skips them and modify the workflow to include that argument.

Reply:
Ok, np, this should be the [fix](https://github.com/deepset-ai/haystack-experimental/pull/55/commits/c636ae823e0795ba354ff85fb9aa093fc617d8ed)